packages/now-cli/src/util/dev/builder.ts

```diff
@@ -359,9 +363,8 @@ export async function executeBuild(
           MemorySize: asset.memory || 3008,
           Environment: {
             Variables: {
-              ...nowConfig.env,
```

Comment:
@TooTallNate Do you know why `nowConfig.env` is here? It seems like this was always captured by `env` below so I removed it.

Reply:
I don't recall specifically, but from the looks of it, it's perhaps because env vars defined by a lambda function build output should override env vars in the `now.json` config?

Reply:
But in this case it would override because `nowConfig.env` was before the `env` below 🤔

Reply:
Actually I'm going to add it back. This is different now that we spoke in slack.